src/app/components/ATIAnalytics/atiUrl/index.ts

```diff
@@ -252,6 +263,7 @@ export const buildATIEventTrackUrl = ({
   advertiserID,
   url,
   detailedPlacement,
+  variant,
```

Comment:
Confusingly, we use the term `variant` for services with dual scripts e.g. serbian with cyrillic script has a variant = cyr

If this is not the intention of the term variant here, can we please rename it to something else, to avoid confusion - perhaps `experimentVariant` or something similar?

Reply:
I saw this as well, but had a look at the utility and apparently it was set for A/B testing 3 years ago https://github.com/bbc/simorgh/blob/275b924d2e75257842b6c5415a332d646beb4731/src/app/lib/analyticsUtils/index.js#L272

Reply:
We've added this here to use as the experiment variant in our use-case because `[getEventInfo()](https://github.com/bbc/simorgh/blob/latest/src/app/lib/analyticsUtils/index.js#L272)` uses this as a variant for A/B testing, it just wasn't previously passed in here.
As the value just needs to be in the right place in the string, I'll rename it in there too and remove the comment to make it clearer